docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
One of those ideas I have on the back burner of my mind is an issue tracker that stores data in a git repo - like github wikis are a repo that you can clone and edit locally. There are a few projects that have tried this, e.g. [Bugs Everywhere](http://bugseverywhere.org/), but it doesn't seem to have taken off. I think it's a combination of it being rather more complicated than it looks at first glance, and regular bug trackers being *good enough* that there's little pressure to work on replacements.